source/docs/software/actuators/overview.rst

```diff
@@ -2,20 +2,21 @@
 
 Actuator Overview
 ==================
-This section discusses the control of motors and pneumatics through speed controllers, solenoids and pneumatics, and their interface with C++ and Java WPILib.
+This section discusses the control of motors and pneumatics through motor controllers, solenoids and pneumatics, and their interface with C++ and Java WPILib.
```

Comment:
This sentence doesn't make sense. Pneumatics are controlled by pneumatics?

Reply:
What about if we just remove the "through...." so it is: This section discusses the control of motors and pneumatics and their interface with C++ and Java in WPILib.